### PR TITLE
Remove bulldog code from sovrin-client; should be in its own repo.

### PR DESCRIPTION
**Pull request by dhh1128**

### CODE REVIEW

**Comment by khagesh** [comment]
@lovesh can you please merge this PR

File: data/pool_transactions_sandbox [hunk 1]
```diff
@@ -1,4 +1,8 @@
...
 {"dest":"13qfUHjTDh3TdRHau93LtkLynwNJnLGoRGh9HQxHuiLs","data":{"services":["VALIDATOR"],"client_port":9724,"node_ip":"52.38.24.189","alias":"EvernymV2","client_ip":"52.38.24.189","node_port":9723},"type":"NODE","txnId":"4e07408562bedb8b60ce05c1decfe3ad16b72230967de01f640b7e4729b49fce","identifier":"2btLJAAb1S3x6hZYdVyAePjqtQYi2ZBSRGy4569RZu8h"}
-{"dest":"BkJAXQsFYsHtqgbMUYSVQ8n3okRvyPfozwWcn4gV538Z","data":{"services":["VALIDATOR"],"client_port":9730,"node_ip":"34.200.79.65","alias":"RespectNetwork","client_ip":"34.200.79.65","node_port":9729},"type":"NODE","txnId":"ef2d127de37b942baad06145e54b0c619a1f22327b2ebbcfbec78f5564afe39e","identifier":"4AdS22kC7xzb4bcqg9JATuCfAMNcQYcZa1u5eWzs6cSJ"}
+{"dest":"AZsp46qSPGjHuNYajiMUb5XcMWCRDuk2xdEv7P5MepSN","data":{"services":["VALIDATOR"],"client_port":9726,"node_ip":"40.121.149.41","alias":"WSECU","client_ip":"40.121.149.41","node_port":9725},"type":"NODE","txnId":"4b227777d4dd1fc61c6f884f48641d02b4d121d3fd328cb08b5531fcacdabf8a","identifier":"CECeGXDi6EHuhpwz19uyjjEnsRGNXodFYqCRgdLmLRkt"}
```
Comment (lovesh) [inline data/pool_transactions_sandbox]: `WSECU`, `BIG`, `FCU` and `BYU` are out of order as of now. Also `RespectNetwork` ip has changed. Please accept whats on the left.

File: data/transactions_live [hunk 2]
```diff
@@ -1,5 +1,5 @@
-|1|6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4a||NYM|9XNVHKtucEZWh7GrS9S8nRWtVuFQwYLfzGD7pQ7Scjtc|||Trustee1||||TRUSTEE|
-|2|6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4b||NYM|5rArie7XKukPCaEwq5XGQJnM9Fc5aZE3M9HAPVfMU2xC|||Steward1||||STEWARD|
-|3|6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4c||NYM|2btLJAAb1S3x6hZYdVyAePjqtQYi2ZBSRGy4569RZu8h|||Steward2||||STEWARD|
-|4|6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4f||NYM|4AdS22kC7xzb4bcqg9JATuCfAMNcQYcZa1u5eWzs6cSJ|||Steward3||||STEWARD|
-|5|6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4f||NYM|6hbecbh36EMK6yAi5NZ9bLZEuRsWFt6qLa2SyMQGXs7H|||Steward4||||STEWARD|
+||7e5d5f22d8c3c68c4532e0a336ba2db3276d12acf7820334bee49b5581d62277||NYM|FYmoFw55GeQH7SRFa37dkx1d2dZ3zUF8ckg7wmL7ofN4|||Steward1||||STEWARD|
```
Comment (lovesh) [inline data/transactions_live]: This file has also changed. Please accept whats on the left.

File: data/transactions_sandbox [hunk 3]
```diff
@@ -1,5 +1,15 @@
 |1|6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4a||NYM|9XNVHKtucEZWh7GrS9S8nRWtVuFQwYLfzGD7pQ7Scjtc|||Trustee1||||TRUSTEE|
```
Comment (lovesh) [inline data/transactions_sandbox]: This file has also changed. Please accept whats on the left.

File: sovrin_client/test/cli/helper.py [hunk 4]
```diff
@@ -191,3 +198,45 @@ def new():
...
+                 totalAvailableClaims=None,
+                 totalSchemas=None,
+                 totalClaimsRcvd=None,
+                 within=None):
+    async def check():
+        assert (totalLinks is None or
+                totalLinks == len(cli.activeWallet._links))
+
+        tac = 0
+        for li in cli.activeWallet._links.values():
```
Comment (lovesh) [inline sovrin_client/test/cli/helper.py]: One line (little faster) alternative, `tac = sum([len(li.availableClaims) for li in cli.activeWallet._links.values()])`

File: sovrin_client/test/cli/helper.py [hunk 5]
```diff
@@ -191,3 +198,45 @@ def new():
...
+        assert (totalClaimsRcvd is None or
+                totalClaimsRcvd == len((await cli.agent.prover.wallet.getAllClaims()).keys()))
+
+    if within:
+        cli.looper.run(eventually(check, timeout=within))
+    else:
+        check()
+
+
+def wallet_starting_state(totalLinks=0,
```
Comment (lovesh) [inline sovrin_client/test/cli/helper.py]: Why do we pass arguments if we dont use them? Is the `return locals()` a temporary thing?

File: sovrin_client/test/cli/test_connect_env.py [hunk 6]
```diff
@@ -5,24 +5,44 @@
 
 
 @pytest.fixture(scope="module")
-def aliceCli(aliceCLI):
+def alice(aliceCLI):
```
Comment (lovesh) [inline sovrin_client/test/cli/test_connect_env.py]: Why be less explicit, what is `alice`, CLI, wallet, agent? Once you have a few more players, it start getting confusing.

**Pull request closed without merging**